src/mainboard/clevo/tgl-u/variants/ns50mu/romstage.c

```diff
@@ -0,0 +1,39 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
Does it need a separate file?

Reply:
Probably not, I'll move it to baseboard code

Reply:
Moved